scripts/build_test.py

```diff
@@ -741,13 +740,12 @@ def test_compiled_js_dir_validation(self):
         with python_utils.open_file(build.TSCONFIG_FILEPATH, 'r') as f:
             config_data = json.load(f)
             out_dir = os.path.join(config_data['compilerOptions']['outDir'], '')
+        # Escape the special charaters in file path.
```

Comment:
Explain why this is needed, making some reference to Windows backslashes and how these get interpreted in regexes.

Reply:
Updated